ush/parsing_namelists_FV3.sh

```diff
@@ -6,6 +6,20 @@
 # shellcheck disable=SC2034
 FV3_namelists(){
 
+# First argument tells us which namelist we're writing:
```

Comment:
We are in the process of replacing this with using a template from the ufs-weather-model, and this change makes it very difficult for us to do that at this time.
I will discuss with @junwang-noaa to find out if there is a way to do it before this part of the PR.

Reply:
Sounds good. Thanks!